proto/viam/app/v1/end_user.proto

```diff
@@ -32,6 +35,25 @@ message RegisterAuthApplicationRequest {
   string org_id = 2;
   repeated string origin_uris = 3;
   repeated string redirect_uris = 4;
+  string logout_uri = 5;
+}
+
+message RegisterAuthApplicationResponse {
+  string application_id = 1;
+  string application_name = 2;
+  string secret = 3;
 }
 
-message RegisterAuthApplicationResponse {}
+message UpdateAuthApplicationRequest {
+  string org_id = 1;
+  string application_id = 2;
+  string application_name = 3;
+  repeated string origin_uris = 4;
+  repeated string redirect_uris = 5;
+  string logout_uri = 6;
```

Comment:
@ohEmily updated this sightly after testing. i realized you need to provide these params because omitting them "resets" the values. basically the application gets fully rewritten with whatever is in the update request. renamed these fields to exclude the `new_` prefix for that reason

Reply:
Ah! You're saying the FusionAuth API will reset these. That's unexpected for me too. Thanks for explaining!